include/circt/Tools/CMakeLists.txt

```
@@ -1 +1,2 @@
 add_subdirectory(circt-lec)
+add_subdirectory(circt-bmc)
```

super nit: please alpha-sort.
```suggestion
add_subdirectory(circt-bmc)
add_subdirectory(circt-lec)
```